bin/configs/java-microprofile-server-multiple-2xx-responses.yaml

```diff
@@ -0,0 +1,8 @@
+generatorName: microprofile
```

Comment:
can we name the generator as "java-microprofile" instead following the convention of other java server generators?

Reply:
done